pkg/globalconfig/global_config.go

```diff
@@ -29,7 +29,7 @@ func init() {
 }
 
 type ProjectInfo struct {
-	UsedHostPorts []string `yaml:"used_host_ports,flow"`
+	UsedHostPorts []string `yaml:"used_host_ports,omitempty,flow"`
```

Comment:
I confirmed that this will hide the `used_host_ports` list when none were requested, but also played around with the logic in CheckAndReserveHostPorts (ddevapp/config.go) that prevents the project info map record from being generated when no relevant port information was provided, I'll comment there as well.